src/templates/tpl-preferences.html

```diff
@@ -3,7 +3,7 @@
 <!-- preferences -->
 <div class="afch afch-form afch-preferences">
 	<div class="details centered">
-		<strong>AFCH v{{version}} / {{versionName}}</strong>
+		<strong>AFCH</strong>
```

Comment:
I think this line can be removed as it only existed for showing the version info.